hivemind/optim/grad_scaler.py

```diff
@@ -21,10 +21,14 @@ class GradScaler(TorchGradScaler):
     :note: if not using reuse_grad_buffers=True, one can and *should* train normally without this class, e.g. using
       standard PyTorch AMP or Apex. This custom GradScaler is more memory-efficient, but requires custom training code.
 
-    GradScaler removes several:
+    hivemind.GradScaler makes 3 modifications to the regular PyTorch AMP:
+
     - bypass .unscale_ and .update calls in order to accumulate gradients over several steps
```

Comment:
This list is not displayed correctly. Same for the list in the `CollaborativeAdaptiveOptimizer`'s docstring.

Reply:
Alas, i have no idea how to fix that :(

Reply:
Okay, it would appear that docutils==0.16 does not have this issue